UKHomeOffice/nmsw-ui
# NMSW-470 p5 Refactor to get/use declarationId from params - supporting docs

# Ticket

NMSW-470

----

## AC

part 5, see https://github.com/UKHomeOffice/nmsw-ui/pull/269

----

## To test

- create a voyage and upload FAL1 and continue to task details page
- click the supporting documents link
- > you should be taken to the upload supporting documents page with the declarationId in the url
- add one or more files
- click upload files (test version of uploading FAL1 only)
- click save and continue
- you should be returned to the task details page

----

## Notes
The following pages will not work as they've not yet been updated

- CYA
- delete
